blueprints/cloud-operations/network-dashboard/src/plugins/discover-cloudsql.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2022 Google LLC
```

Comment:
this could be done in cai-compute discovery, but as you said it also makes sense to have a dedicated discovery module for PSA resources; I would not add more than one though :)